feat: Include references to new Revenue analytics product in new-to-posthog docs (#11352)
PostHog/posthog.com: contents/docs/new-to-posthog/revenue.mdx

## Revenue reporting

With revenue logged, you need a way to see it. There are two options here.

### Revenue analytics dashboard

> Revenue analytics is one of our most recent products – and as such is still in beta. It lets you see the revenue generated by your business over time. Not only that, it will also let you see what your growth and LTV looks like.

Reviewer: ```suggestion
Revenue analytics is now in beta! It enables easy reporting of revenue generated over time. You'll also get a new lens on revenue growth and lifetime customer value.
```

<ProductScreenshot
    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2025_04_22_at_15_18_52_079665efa3.png" 
    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2025_04_22_at_15_18_35_71ced993e5.png"
    alt="Revenue analytics dashboard"
    classes="rounded"
/>

> We'll keep extending this product to give you more and more out-of-the-box insights.

Reviewer: ```suggestion
Revenue analytics is a work-in-progress, so stay tuned for additional insights.
```

> ### Custom trends insights

Reviewer: ```suggestion
### Build a custom trend insight
```
